templates/node--islandora-rdm-dataset--search-result.html.twig

```diff
@@ -0,0 +1,18 @@
+
+{% set created_date = node.getCreatedTime|format_date('umami_dates') %}
+{{ attach_library('classy/node') }} {# apply classy theme to node}
+                                    
+{# Format Creator (PublicationYear): Title. Publisher. (resourceTypeGeneral). Identifier #}
+<h4>{{ node.field_rdm_creator.0.entity.field_rdm_personal_name.value }} 
```

Comment:
This should be an h2 since you shouldn't use a lower level than you need to.  We can set styling for an h2 below a search result set if we need to make it smaller.